binding/default_validator.go

```diff
@@ -85,13 +83,14 @@ func (v *defaultValidator) validateStruct(obj interface{}) error {
 // or struct level validations. See validator GoDoc for more info -
 // https://pkg.go.dev/github.com/go-playground/validator/v10
 func (v *defaultValidator) Engine() interface{} {
-	v.lazyinit()
+	v.lazyInit()
 	return v.validate
 }
 
-func (v *defaultValidator) lazyinit() {
-	v.once.Do(func() {
+// lazyInit initialize the validate Settings, no need to use sync.once
+func (v *defaultValidator) lazyInit() {
+	if v.validate == nil {
```

Comment:
I'm not sure if this is an improvement. I believe `sync.Once` was used to avoid race conditions. This version seems kind of safe, too, but it might call `validator.New()` multiple times unnecessarily. I don't see the point of eliminating `sync.Once`.